### PR TITLE
New package: ForgQi.biliup-rs version 0.1.15

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

### CODE REVIEW

File: manifests/f/ForgQi/biliup-rs/0.1.15/ForgQi.biliup-rs.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ForgQi.biliup-rs
+PackageVersion: 0.1.15
+InstallerType: zip
+ReleaseDate: 2022-10-03
+Installers:
```
Comment (stephengillie) [inline manifests/f/ForgQi/biliup-rs/0.1.15/ForgQi.biliup-rs.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/f/ForgQi/biliup-rs/0.1.15/ForgQi.biliup-rs.installer.yaml:8]: The package installs, but requires this dependency to run:
image

**Comment by Trenly** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**